k-distribution/tests/regression-new/kprove-java/rhs-vars-broken-spec.k.out

```diff
@@ -1,15 +1 @@
 #Top
-[Warning] Critical: Found variable _ on right hand side of rule, not bound on
-left hand side. Did you mean "?_"?
-Note: this warning will become an error in subsequent releases.
-	Source(rhs-vars-broken-spec.k)
-	Location(10,20,10,21)
-[Warning] Compiler: Variable 'A' defined but not used. Prefix variable name
-with underscore if this is intentional.
-	Source(rhs-vars-broken-spec.k)
-	Location(12,20,12,21)
-[Warning] Critical: Found variable A on right hand side of rule, not bound on
-left hand side. Did you mean "?A"?
-Note: this warning will become an error in subsequent releases.
-	Source(rhs-vars-broken-spec.k)
-	Location(12,20,12,21)
```

Comment:
It seems to me this should be part of the test? Seems it's testing broken variables?

Reply:
@traiansf you are the author of this test. Is this what you had in mind?